src/components/KInput/KInput.vue

```diff
@@ -339,27 +342,28 @@ export default {
     }
 
     &.icon-start {
-      padding-left: calc($kInputMediumSizingY + var(--spacing-xs, spacing(xs)) + $kInputMediumIconSize) !important; // account for icon offset and width
+      padding-left: calc($kInputMediumSizingX + var(--spacing-xs, var(--kui-space-40, $kui-space-40)) + $kInputMediumIconSize) !important; // account for icon offset and width
       ~ .input-icon {
-        left: $kInputMediumSizingY;
+        left: $kInputMediumSizingX;
```

Comment:
question: variables here changed from using Y to X -- was that intentional?

Same comment applies to all of the blocks below

Reply:
Yes, that is intentional - those variables got renamed because they were wrong before (notice how we were using Y variable for `left` property - which is X axis).